ci-operator/step-registry/ipi/conf/vsphere/folder/ipi-conf-vsphere-folder-commands.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+# shellcheck disable=SC1091
+source "${SHARED_DIR}/vsphere_context.sh"
+echo "$(date -u --rfc-3339=seconds) - Configuring govc exports..."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+FOLDER="/$vsphere_datacenter/vm/$vsphere_datacenter"
+SUB_FOLDER="$FOLDER/qe-folder"
```

Comment:
Suggest to use cluster name (`${NAMESPACE}-${UNIQUE_HASH}`) to name the folder to get an uniq id.

Reply:
OK, so how to pass it to folder check step, write a folder.txt file or use env? What would you like?